File: 백준/Bronze/2562. 최댓값/최댓값.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n').map(x => Number(x));
let maxNum = input[0];
let num = 0;
for (let i = 1; i < 9; i++) {
    if (maxNum < input[i]) {
        maxNum = input[i];
        num = i;
    }
}
console.log (maxNum);
console.log (num + 1);

File: 백준/Bronze/2562. 최댓값/최댓값.py
nums = [int(input()) for i in range(9)]

max_num = nums[0]
index = 0 + 1

for i in range(len(nums)):
    if max_num < nums[i]:
        max_num = nums[i]
        index = i + 1

print(max_num)
print(index)